README.md

```diff
@@ -212,6 +230,63 @@ $ curl localhost:9200/_cat/indices
 green open ks-logstash-log-2021.04.06 QeI-k_LoQZ2h1z23F3XiHg  5 1 404879 0 298.4mb 149.2mb
 ```
 
+### Fluentd 
+
+Fluentd acts as a forward log layer to collect logs from fluentbit and other Apps. 
```

Comment:
=>
Fluentd acts as a log forward layer that receives logs from fluent-bit or other Apps through the network.